monaco.d.ts

```diff
@@ -4807,13 +4807,13 @@ declare namespace monaco.languages {
         /**
          * A range of text that should be replaced by this completion item.
          *
-         * Defaults to a range from the start of the [current word](#TextDocument.getWordRangeAtPosition) to the
+         * Defaults to a range from the start of the [current word]{@link ITextModel.getWordAtPosition} to the
```

Comment:
I believe it actually uses [`getWordUntilPosition`](https://github.com/microsoft/vscode/blob/master/src/vs/editor/contrib/suggest/suggest.ts#L125)

(Thanks for making this PR btw.. I came here to do the exact same thing)